content/blog/urbit-is-for-communities.md

```diff
@@ -0,0 +1,97 @@
+
+This is more or less the entire technical value proposition of Urbit: to build a system that’s simple enough that it requires almost no maintenance. Our approach to building software makes all of the above problems easily solvable.
+
+Perhaps more importantly we have a direct incentive to solve these problems. Tlon funds Urbit’s development by selling addresses and providing hosting. And the only reason anyone pays us for these things is because we make Urbit as directly useful for you as possible.
+
+That means we have to solve the problems inherent to the legacy internet. There can’t be any sense of suspicion or uncertainty in our new system. Your Urbit has to be something you can really trust and attach to. This is why your Urbit lets you run or write whatever code you like, and why we’re focused on making sure the data you put into your Urbit lasts forever: it won’t disappear when some company dies.
+
+But even with a system that fixes all of these issues by default, it only matters if the system is usable and understandable. What we really want is a digital world that’s generated by the communities that inhabit it — which means it has to be generated by people with no technical knowledge.
+
+
+## Communities are built by people
+
+In the physical world, you can arrange the furniture in your house, or go to Ikea, or build your own furniture, or figure out how to build a house from scratch. By taking your preferred approach to design, you contribute to the shape of your city. The world around us is a wonderfully loose collaboration between the more technically oriented architects and planners, and the everyday homeowners and residents.
+
+The digital world is, of course, nothing like this. Even software developers are stuck with the current system. None of our existing apps or services can connect to one another, so whether you’re a user or a developer, you’re faced with the daunting task of replacing everything all at once if you want a better system. But nobody can do that.
+
+Urbit provides an experience with less distance between ordinary use and total customization. To start, we’ve created Ikea-like, Lego-like and woodshop-like levels of usability. What we’ve built isn’t nearly as complex as the physical world of course, but it’s a start.
+
+For most people, customizing your Urbit should be a matter of picking from an available suite of modules. Not unlike going to Ikea. For some people, recombining interfaces and data sources within Ubit should feel like playing with Legos. And for others, who want to dive in and write code, it’s the same as going into a shop full of high quality tools.
+
+We’re not quite to this level of sophistication, but we’re on the path. Let’s talk about where we are and where we’re going.
+
+
+## OS 1
+
+Our roughest attempt to satisfy this goal is hovering on the horizon. We call it OS 1.
```

Comment:
Inadvertently a self-own by saying it's our roughest ever release; maybe "The foundations for satisfying this goal are hovering on the horizon" @galenwp

Reply:
I can see where you're coming from where 'rough' is translated as a bad/negative word, but I read 'rough' in the sense of 'rough draft', which is basically a positive thing if you believe in sharing rough drafts with people

Reply:
I'd be fine with roughest -> rough